lib/helpers/is-interactive-element.js

```diff
@@ -2,9 +2,11 @@
 
 const ast = require('./ast-node-info');
 
+// Spec: https://html.spec.whatwg.org/#interactive-content
 const INTERACTIVE_TAG_NAMES = new Set([
+  'a',
+  'audio',
```

Comment:
This seems incorrect. `audio` should only be added if `controls` attribute is present. See spec link / image above.